guides/quickstart.md

```diff
@@ -0,0 +1,107 @@
+# Quickstart: Flash Nerves Livebook
+
+This guide shows you how to download and flash the latest Nerves Livebook
+firmware in one simple flow. By the end, you’ll have Livebook running on your
+device and be ready to explore tutorials in your browser.
+
+---
+
+## Prerequisites
+
+1. A supported board (Raspberry Pi Zero/Zero W, Pi 4, BeagleBone Black, etc.)
+2. A micro‑SD card and an SD‑card reader (built‑in slot or USB adapter)
+3. On your host machine:
+   - bash (4+)
+   - curl
+   - fwup (requires `sudo`; see https://github.com/fwup-home/fwup)
+
+---
+
+## Clone the repository
+
+```bash
+git clone https://github.com/nerves-livebook/nerves_livebook.git
+cd nerves_livebook
+```
+
+---
+
+## Flash with one command
+
+Make the helper script executable:
+
+```bash
+chmod +x scripts/flash_livebook.sh
+```
+
+Then run:
+
+```bash
+./scripts/flash_livebook.sh
+```
+
+You will be prompted to:
+
+- Select your `MIX_TARGET` (e.g. `rpi0`, `rpi4`, `bbb`)
+- Enter WiFi SSID and passphrase
+- Optionally set a custom serial number
+- Insert your SD card when asked
+
+The script will download the latest `.fw`, provision it, wait for your card,
+and invoke `sudo fwup …` for you.
+
+> #### Single‑command installer
```

Comment:
Do you think this section should come first? It seems easier than cloning the repository for a first time user just trying to evaluate the project.